modules/core/src/main/java/org/apache/ignite/internal/management/baseline/BaselineTaskResult.java

```diff
@@ -171,37 +175,46 @@ private static Collection<InetAddress> resolveInetAddresses(ClusterNode node) {
     /**
      * Constructor.
      *
-     * @param active Cluster state.
+     * @param clusterState Cluster state.
      * @param topVer Current topology version.
      * @param baseline Current baseline nodes.
      * @param servers Current server nodes.
      * @param remainingTimeToBaselineAdjust Time to next baseline adjust.
      * @param baselineAdjustInProgress {@code true} If baseline adjust is in progress.
      */
     public BaselineTaskResult(
-        boolean active,
+        ClusterState clusterState,
         long topVer,
         Collection<? extends org.apache.ignite.cluster.BaselineNode> baseline,
         Collection<? extends org.apache.ignite.cluster.BaselineNode> servers,
         BaselineAutoAdjustSettings autoAdjustSettings,
         long remainingTimeToBaselineAdjust,
         boolean baselineAdjustInProgress) {
-        this.active = active;
+        this.clusterState = clusterState;
         this.topVer = topVer;
         this.baseline = toMap(baseline);
         this.servers = toMapWithResolvedAddresses(servers);
         this.autoAdjustSettings = autoAdjustSettings;
         this.remainingTimeToBaselineAdjust = remainingTimeToBaselineAdjust;
         this.baselineAdjustInProgress = baselineAdjustInProgress;
+
+        active = !(clusterState == ClusterState.INACTIVE);
     }
 
     /**
-     * @return Cluster state.
+     * @return Cluster activated flag.
      */
     public boolean isActive() {
         return active;
```

Comment:
It seems we don't need the `active` field at all, `return clusterState.active()`